### ISSUE TITLE
100% accuracy with poor predictions on finetuned model

### ISSUE THREAD

**Issue by sultanovazamat** [clovaai/deep-text-recognition-benchmark#435]
Hi!

Has anyone faced the issue of ideal accuracy already on the 20th iteration of FT, but with poor results?

<img width="894" alt="Image" src="https://github.com/user-attachments/assets/ec7625f8-92c0-443e-9549-c01aed490547" />

This is my training command:

`!python train.py --train_data /content/drive/MyDrive/lmbd_output_train --valid_data /content/drive/MyDrive/lmbd_output_val --select_data "/" --batch_ratio 1 --Transformation TPS --FeatureExtraction ResNet --SequenceModeling BiLSTM --Prediction Attn --batch_size 256 --data_filtering_off --workers 0 --batch_max_length 80 --num_iter 1000 --valInterval 10 --sensitive --saved_model /content/drive/MyDrive/TPS-ResNet-BiLSTM-Attn-case-sensitive.pth --data_filtering_off --FT --adam --imgW 180 --PAD`

I am using small datasets: train - 3000 imgs and val - 1500 imgs to set up the training correctly and then run the training on the huge dataset.
